File: day-06/2.ts
export {};

const input = await Deno.readTextFile("./input.txt");

for (let i = 14; i < input.length; ++i) {
  const s = input.substring(i - 14, i);
  if (new Set([...s]).size === s.length) {
    console.log(i);
    break;
  }
}

File: day-01/2.ex
File.read!("./input.txt")
|> String.split("\n\n")
|> Enum.map(fn elf ->
  elf
  |> String.split("\n")
  |> Enum.map(fn elf -> elf |> String.to_integer() end)
  |> Enum.sum()
end)
|> Enum.sort(:desc)
|> Enum.slice(0..2)
|> Enum.sum()
|> IO.inspect()
